### PR TITLE
Release v0.5.0

### PR DESCRIPTION
**Pull request by etungsten**
This commit bumps the version from v0.4.0 to v0.5.0 and adds a changelog for the admin container.

**Issue number:**
N/A

**Description of changes:**
Prepares for v0.5.0 release of the bottlerocket admin container.

**Testing done:**
N/A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# CHANGELOG
+
+## 0.5.0
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
# 0.5.0
```
I recommend we change this to match the bottlerocket-os changelog.  No need to call out that it's a changelog again.

(Would also have to remove a "#" from 0.4.0 below.)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+* Improvement - [sheltie: use /proc to find the bash binary](https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/8) (#8, @samuelkarp)
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Use /proc to find the bash binary in sheltie. ([#8])

[#8]: https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/8
```

I recommend keeping the same voice and format as bottlerocket-os.

**Comment by etungsten** [comment]
Addresses @tjkirch 's comments.

File: CHANGELOG.md (hunk 2 above)
Comment (tjkirch) [reply CHANGELOG.md]: Do you not need the PR link?  I thought we did in standalone markdown documents like changelogs.

> `[#8]: https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/8`

Comment (etungsten) [reply CHANGELOG.md]: Sorry I missed that!

**Comment by etungsten** [comment]
Fixed missing PR link.

**Review by tjkirch — APPROVED** [review]

**Review by zmrow — APPROVED** [review]
📟

**Pull request merged**